images/metering-ansible-operator/roles/meteringconfig/tasks/reconcile.yml

```diff
@@ -12,6 +12,78 @@
     hiveStorageType: "{{ meteringconfig_spec_overrides | json_query('storage.hive.type') }}"
   when: hiveStorageType is undefined or hiveStorageType not in ['s3', 'sharedPVC', 'hdfs']
 
+- name: Check if we need to create a Presto TLS secret using the user-populated fields
```

Comment:
Everything you've added to this file should go into it's own task file.

Reply:
Maybe rename `configure_tls.yml` to `configure_presto_tls.yml` and put these tasks into a `configure_tls.yml` task file.

Reply:
Good idea, I'll update the conditional on the `configure_tls.yml` to rely on the top-level `meteringconfig_spec.tls.enabled` flag.